.idea/Working version/algorithms.js

```diff
@@ -0,0 +1,472 @@
+/**
+ * Created by tsodergren on 6/5/17.
+ */
+
+
+function constructEdges() {
+
+
+    var radiusChange = arguments.length ? arguments[0] : 0;
+
+    //squared coverage diameter and annulus of uncertainty
+    var sqDiameter = 4 * Math.pow(complexRadius, 2);
+    var sqDiameterMin = 4 * Math.pow(complexRadius-dataRadius, 2);
+    var sqDiameterMax = 4 * Math.pow(complexRadius+dataRadius, 2);
+
+
+    if (!a_edges) {
```

Comment:
Because a_edges is only populated on file load, clearing the screen completely breaks it. For example, I have  attached a picture of what happened when I cleared the screen and added nodes. It never generated edges because a_edges was just zeros. Faces appeared, but no edges. Only after increasing the coverage radius to be huge did it find one edge when there should have been way more.
<img width="1416" alt="screen shot 2017-07-06 at 10 00 16 pm" src="https://user-images.githubusercontent.com/11620565/27943812-ee788b58-6296-11e7-890b-95bfaecb4cab.png">